includes/wikia/models/PromoImage.class.php

```diff
@@ -34,39 +29,18 @@ public function __construct($type, $dbName = null) {
 		$this->removed = false;
 	}
 
-	public function isType($type){
+	private function isType($type){
 		return $this->type === $type;
 	}
 
-	public function isAdditional(){
-		return in_array($this->type, self::listAllAdditionalTypes());
-	}
-
-	public function isValid(){
-		return !$this->isType(self::INVALID) and $this->isCityIdSet();
-	}
-	public function wasRemoved(){
-		return $this->removed;
-	}
-
-	public function getType(){
-		return $this->type;
-	}
-
-	public function setDBName($dbName) {
-		$this->dbName = $dbName;
-		$this->city = null;
-		return $this;
-	}
-
-	public function getDBName(){
+	private function getDBName(){
 		if (empty($this->dbName) and !empty($this->cityId)) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace "and" with "&&". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS2010)